index.js

```diff
@@ -12,11 +12,15 @@ const { log } = console;
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use('/api/v1', routes);
```

Comment:
Why was this line removed? And what will be the alternative for specifying a base version path for the routes?

Reply:
@LukasChiama Thanks for the feedback. While this is the base path for the app's routes, and I don't think it is the base path for the app's root. Going by the way this was implemented, `/api/v1/` also applies to the root, which I feel is wrong (see [this article](https://www.h3xed.com/web-development/what-is-the-document-web-root-directory-for-a-website))